packages/vow/src/watch-utils.js

```diff
@@ -111,7 +111,7 @@ export const prepareWatchUtils = (zone, watch, makeVowKit) => {
           }
           resolver.resolve(harden(results));
         },
-        onRejected(value, { id }) {
+        onRejected(value, { id, index: _index }) {
```

Comment:
stray?

Reply:
Nope it's there to satisfy typescript, in particular https://github.com/Agoric/agoric-sdk/blob/6b62e09cfe8fc348e571c86cf17a96c0bfccf133/packages/vow/src/watch-utils.js#L65

Now that this PR correctly [plumbs the args in the rejection handler in the type definitions](https://github.com/Agoric/agoric-sdk/blob/6b62e09cfe8fc348e571c86cf17a96c0bfccf133/packages/vow/src/types.js#L84-L87), the extra property check of TS kicked in with the type of the args which were inferred to the narrower / common part of the fulfillment and rejection handler. Maybe there is a better way to type `watch` to avoid this gotcha?